File: 6labs/c#/TextContainer.cs
public class TextContainer
{
    private List<TextLine> lines = new();

    public void AddLine(TextLine line) => lines.Add(line);
    public void RemoveLine(int index) => lines.RemoveAt(index);
    public void ShowAll()
    {
        foreach (var line in lines)
            Console.WriteLine(line.GetContent());
    }
}

File: 6labs/ts/TextContainer.ts
import { TextLine } from "./TextLine";

export class TextContainer {
    private lines: TextLine[] = [];

    addLine(line: TextLine): void {
        this.lines.push(line);
    }

    removeLine(index: number): void {
        this.lines.splice(index, 1);
    }

    showAll(): void {
        this.lines.forEach(line => console.log(line.getContent()));
    }
}
